apps/admin/_services/validation.service.ts

```diff
@@ -109,7 +114,13 @@ export class ValidationService extends BaseService {
 
     return {
       rule: ValidationRuleEnum.NoInnovationsSupportedOnlyByThisUser,
-      valid: innovationSupportedOnlyByUser.length === 0
+      valid: innovationSupportedOnlyByUser.length === 0,
+      ...(innovationSupportedOnlyByUser.length && {
+        details: {
+          unit: innovationSupportedOnlyByUser[0]?.innovationSupports[0]?.organisationUnit?.name,
```

Comment:
Could this just use `role.organisationUnit?.name` instead of needing the extra column in the select() above?

Reply:
We can, pretty much the same thing though but I like your idea better